streaming-binlog/pom.xml

```diff
@@ -136,6 +137,23 @@
       <version>4.2.0</version>
       <scope>compile</scope>
     </dependency>
+    <!-- OpenMRS -->
+    <dependency>
+      <groupId>org.openmrs.api</groupId>
+      <artifactId>openmrs-api</artifactId>
+      <version>${openmrsPlatformVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.openmrs.module</groupId>
+      <artifactId>atomfeed-api</artifactId>
```

Comment:
How much code is the feed configuration related classes? I took a quick look at `FeedConfigurationServiceImpl` and it seems fairly straightforward and thought maybe it is easier to duplicate that logic directly, what do you think? (not in this PR but for future if that is the only way to get rid of openmrs-api dep).

Reply:
Yes, I was actually contemplating of doing exactly this but we can do this in another PR. I have created this ticket

Reply:
Here us the ticket #36

Reply:
Thanks; resolving.